third_party/3/pkg_resources/__init__.pyi

```diff
@@ -202,7 +202,19 @@ class IMetadataProvider:
 
 class ResolutionError(Exception): ...
 class DistributionNotFound(ResolutionError): ...
-class VersionConflict(ResolutionError): ...
+
+class VersionConflict(ResolutionError):
+    @property
+    def dist(self) -> List[Any]: ...
+    @property
+    def req(self) -> List[Any]: ...
+    def report(self) -> str: ...
+    def with_context(self, required_by: Dict[str, Any]) -> ContextualVersionConflict: ...
```

Comment:
This can also be a `VersionConflict` if `required_by` is empty. Since `VersionConflict` is the base class, we could just use this for the time being.

Reply:
Added an `if` condition to check. Not sure how apt that is though.